script.js

```diff
@@ -0,0 +1,34 @@
+const hambutton = document.getElementById('menu-harmbuger');
+const main = document.querySelector('.container');
+const menu = document.querySelector('.modal-container');
+
+const hamburgerTwo = document.getElementById('modal-harmburger');
+
+function openMenu() {
```

Comment:
- [ ] Your mobile menu is not working as intended. When the hamburger menu is clicked to launch the mobile menu for the first time and then the cancel button of the mobile menu overlay is clicked, the mobile menu does not open on subsequent attempts when the hamburger menu is clicked. Kindly ensure to improve upon this aspect
- [ ] Your mobile menu is not displaying fully when the hamburger menu is clicked for the first time. Kindly ensure that the mobile menu overlay fits a particular mobile screen with size

Mobile menu overlay not displaying correctly: 
![image](https://user-images.githubusercontent.com/12745474/132316920-1e82799d-09ce-4a71-9314-204698444c24.png)

Reply:
Hey Robert your last statement is not clear to be can you hight more on that

Reply:
When the hamburger is clicked the popup does not fill the entire mobile screen, it only covers some portions of the mobile screen, hence your mobile popup menu is not responsive. Kindly endeavor to improve upon that aspect